SPARKCONF.md

```diff
@@ -0,0 +1,155 @@
+##Spark Configuration
+
+oni-ml main component uses Spark and Spark SQL to analyze network events and produce a list of least probable events
+or most suspicious. 
+
+Before running oni-ml users need to configure the component. Here are our recommended settings.  
+
+### Yarn tuning
+
+oni-ml Spark application has been developed and tested on CDH [Yarn](http://spark.apache.org/docs/latest/running-on-yarn.html) 
+clusters. Careful tuning of the Yarn cluster may be necessary before analyzing large amounts of data with oni-ml.
+
+For small data sets, perhaps a couple of hundreds of gigabytes of CSV data, default _Yarn_ configurations should be enough
```

Comment:
"perhaps a couple of hundreds of gigabytes of CSV data"

do we support CSV data any more? where is the csv data coming from? I thought we moved it all to parquet